examples/ZMQPP/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ZMQPP)
+
+hunter_add_package(ZMQPP)
+
+find_package(ZMQPP CONFIG REQUIRED)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
@ruslo please clarify! You wanna add c++11 flags in example `CMakeLists.txt` or I can raise error message but currently I have to check the `CMAKE_CXX_FLAGS` variable instead of `CMAKE_CXX_STANDARD` variable?

Now as I see the `CMakelists.txt` of zmqpp the building of ZMQPP will failed if C++11 is not enabled (on linux).